digicampipe/io/containers.py

```diff
@@ -44,6 +47,38 @@
            'DataContainer']
 
 
+class CameraEventType(IntFlag):
+    '''
+    I do not know where this comes from, but I found it here:
+    https://github.com/cta-sst-1m/digicampipe/issues/244
+
+    ALG0       (0x1)     /* algorithm 0 trigger - PATCH7 */
+    ALG1       (0x2)     /* algorithm 1 trigger - PATCH19 */
+    ALG2       (0x4)     /* algorithm 2 trigger - MUON */
+    INTRNL     (0x8)     /* internal or external  */
+    EXTMSTR    (0x10)    /* unused (0) / external (on master only)  */
+    BIT5       (0x20)    /* unused (0)  */
+    BIT6       (0x40)    /* unused (0)  */
+    CONT       (0x80)    /* continuous readout marker  */
+    CSP_MUON   (0x10000) /* camera server detected muon*/
+    CSP_HILLAS (0x20000) /* camera server computed Hillas parametrs */
+
+    I removed the EVTTYPE, since this is clear from the class name
+    I removed DIGICAM since this is clear from the project.
+    '''
+    UNKNOWN = 0x0
+    ALG0 = 0x1
+    ALG1 = 0x2
+    ALG2 = 0x4
+    INTRNL = 0x8
+    EXTMSTR = 0x10
+    BIT5 = 0x20
+    BIT6 = 0x40
+    CONT = 0x80
+    CSP_MUON = 0x10000
+    CSP_HILLAS = 0x20000
```

Comment:
do we really like these names? `INTRNL`??? really? I assumed someone would speak up here.

at least I can tell you that I misspelled this variable already like:
 - INTERNL
 - INTRNAL
 - INTERNAL

because it takes actually longer to remember which letters had been removed than typing the correctly spelled english word internal.

Reply:
See my comment

Reply:
ALG0 --> CLUSTER7
ALG1 --> CLUSTER19
ALG2 --> MUON
INTRNL --> INTERNAL
BIT5 --> No clue what it is
BIT6 --> No clue what it is
CONT --> No clue what it is
CSP_MUON --> No exact clue what it is
CSP_HILLAS --> No exact clue what it is

Reply:
from Jurek:
CSP_HILLAS flag is set whenever:
* NO CSP_MUON is set (muon detection is done before hillas)
* number of pixels  passing tailcuts if >=3 (tailcut thresholds are
independent for muon and hillas)
* ellipsoid fit algorithm converges (i.e. no singular matrix)

Reply:
did we resolve this?